develop install fails

I'm trying a dev install on macOS 10.11.6.
Tried both brew npm (which install nodejs and npm) and the pkg install from the linked Node.js website in the docs. Both times I get this, after doing `pip install -e .`:

```bash
   npm ERR! Darwin 15.6.0
    npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "build"
    npm ERR! node v6.9.2
    npm ERR! npm  v3.10.9
    npm ERR! code ELIFECYCLE
    npm ERR! nbdime-webapp@0.1.0 build: `webpack`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the nbdime-webapp@0.1.0 build script 'webpack'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the nbdime-webapp package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     webpack
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs nbdime-webapp
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls nbdime-webapp
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     /Users/klay6683/Dropbox/src/nbdime/nbdime/webapp/npm-debug.log
    rebuilding js and css failed
    missing files: ['/Users/klay6683/Dropbox/src/nbdime/nbdime/webapp/static/build/nbdime.js']
```
The requsested npm-debug.log is here:
https://gist.github.com/f24c03a5976ac9cd7aaf376be04f576b

I have zero experience with `npm` so I hope this is all you need as info?

--- a/nbdime/webapp/package.json
+++ b/nbdime/webapp/package.json
@@ -21,7 +21,7 @@
     "source-map-loader": "^0.1.5",
     "style-loader": "^0.13.1",
     "ts-loader": "^1.2.1",
-    "typescript": "^2.0.3",
+    "typescript": "~2.0.3",
     "url-loader": "^0.5.7",
     "watch": "^1.0.1",
     "webpack": "^1.13.1"

